docs/sources/operations/query-acceleration-blooms.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Query Acceleration with Blooms (Experimental) 
+menuTitle: Query Acceleration with Blooms (Experimental) 
```

Comment:
This is a bit too long for a menuTitle. Since the topic is marked as experimental, I don't think we need to mention it in the navigation.
```suggestion
menuTitle: Query Acceleration with Blooms 
```